specifyweb/frontend/js_src/lib/components/SchemaConfig/UniquenessRules.tsx

```diff
@@ -298,33 +302,34 @@ function UniquenessRuleRow({
         )}
         {rule.fields.map((field, index) => (
           <Input.Text
```

Comment:
(optional) could use CustomSelectElement here to be consistent with the query builder and workbench mapper?

i.e for an example of read-only usage of CustomSelectElement, could see this validation message that pops up in workbench mapper when some required field is unmapped:

![Screenshot 2024-01-05 at 19 40 39](https://github.com/specify/specify7/assets/40512816/d5cb3b53-92f2-4ab2-b579-568f5a46d867)